README.md

```diff
@@ -53,7 +53,7 @@ is being translated to your local ipv6 address rather than the ipv4 address. If
 
 ## Faster plugin development
 
-You could run `mvn install` or `mvn jetty:run` every time to incrementally develop the `activemq-artemis-extension` 
+You could run `mvn install` or `mvn jetty:run` every time to incrementally develop the `artemis-extension` 
```

Comment:
Did I see comment recently that it needs a different jetty plugin command these days?